### DISCUSSION TITLE
Insert products grid (like MailChimp)

### DISCUSSION THREAD

**Discussion by SuperVentureStudio** [GrapesJS/grapesjs#4124] [Q&A]
Hi,

Is it possible to insert a products grid like MailChimp?

https://mailchimp.com/help/use-product-content-blocks/

Product content blocks can be basically 2 types:

1. Specific product / list of products from a category
2. Dynamic 

The data will come from the CMS, but the editor needs to have a way to insert that kind of block.

Happy to sponsor the development of this block if it's not currently available.

Thank you

**Comment by artf** [comment]
Well, as your question is so generic, I guess you have 2 options:
1. You read the GrapesJS documentation and do it yourself.
2. You hire someone to do the work.

**Reply by SuperVentureStudio** [reply]
Hi @artf ,

Thanks for making GrapeJS!

Where can I find someone to hire to do this?

Btw I don't think it's so generic, it's a specific need for a new type of block on the editor.

The block will add a grid (or just one single) product with it's image, name, price and link.

The data for the product will come from CMS, but it can be added in an example JSON object at first.

This can be used both in emails (ex: product recommendations, abandoned cart recovery) or landing pages where you want to list products for sale.

Thank you

**Reply by artf** [reply]
Sorry, I meant I can't add to the core a component for your specific case, everybody else will have other requirements, different UI needs, etc. This is why we have plugins and Components API, where you can create your own logic and UI for components.

About hiring people you could start from this post https://github.com/artf/grapesjs/issues/2686 or maybe posting a job offer on the [discord channel](https://discord.gg/QAbgGXq)
